Make documentation as the sip-first (#1864)
VirtusLab/scala-cli: website/docs/reference/scala-command/index.md

 - We do not want to overwhelm users with multiple options and commands
 - We want to make sure that the commands we add to `scala` are stable so once we commited to supporting given option it may be hard to remove it later

> That is why we built in a mechanism to limit the commands, options, directives in Scala CLI by default. However, it's still possible to enable all features by explicitly passing the `--power` flag on the command line, or by setting it globally by running: 

Reviewer: ```suggestion
That is why we built in a mechanism to limit the commands, options, directives in Scala CLI by default. However, it's still possible to enable all features by explicitly passing the `--power` flag on the command line, or by setting it globally running: 
```

```bash ignore
scala-cli config power true
```

> Mainly for SIP submission we have prepared a pages with supported [options](./cli-options.md), [commands](./commands.md) and [using directives](./directives.md) when running Scala CLI with limited functionalities.

Reviewer: ```suggestion
To check which options, commands and directives are supported when running Scala CLI with limited functionalities, refer to [options](./cli-options.md), [commands](./commands.md) and [using directives](./directives.md), respectively.
```
